InventoryService/InventoryService/DataAccess/Implementation/Inventory.cs

```diff
@@ -0,0 +1,38 @@
+﻿using InventoryService.AppDbContext;
+using InventoryService.DataAccess.Interface;
+using InventoryService.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace InventoryService.DataAccess.Implementation
+{
+    public class Inventory : IInventory
+    {
+        private readonly DBContext _dbContext;
+        public Inventory(DBContext context)
+        {
+            _dbContext = context;
+        }
+
+        public void addProduct(Models.Inventory inventory)
+        {
+            if (inventory == null) throw new ArgumentNullException(nameof(inventory));
```

Comment:
can be simplified by ArgumentNullException.ThrowIfNull(inventory, nameof(inventory))